actions/call_api.py

```diff
@@ -1,10 +1,22 @@
 from lib.actions import ZabbixBaseAction
 from zabbix.api import ZabbixAPI
 
+import six
+
 
 class CallAPI(ZabbixBaseAction):
     def run(self, api_method, token, **params):
         # Initialize client object to connect Zabbix server
+
+        # Python doesn't let you modify the length of a dict or list while iterating
+        iterate_params = params.copy()
+
+        # Look for Empty strings and None values, but allow False
+        # If None or '', remove it from params.
```

Comment:
This description is different from actual processing, you should also fix it.

Reply:
Removed in lieu of previous logic change suggestion.